msgid ""
msgstr ""
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: lokalise.com\n"
"Project-Id-Version: Backend, python - mono\n"
"Last-Translator: lokalise.com\n"
"Language-Team: lokalise.com\n\n"
"Language: fr_CA\n"

#: messaging/resources/messaging.py:75
msgid "available_credits_error"
msgstr "Erreur lors de la récupération des crédits de message disponibles"

#: messaging/resources/messaging.py:516
msgid "cannot_attach_asset_belonging_to_another_user"
msgstr "Impossible de joindre un actif appartenant à un autre utilisateur"

#: messaging/resources/messaging.py:535
msgid "cannot_attach_asset_to_message_asset_state_is_not_complete"
msgstr "Impossible de joindre l'actif au message, l'état de l'actif n'est pas COMPLET"

#: messaging/resources/messaging.py:556
msgid "cannot_attach_asset_to_multiple_messages_at_once"
msgstr "Impossible de joindre un fichier à plusieurs messages à la fois"

#: messaging/resources/messaging.py:71
msgid "cannot_charge_customer_error"
msgstr "Impossible de facturer le client."

#: messaging/resources/messaging.py:72
msgid "cannot_charge_customer_stripe_error"
msgstr "Impossible de facturer le client, erreur Stripe"

#: messaging/resources/messaging.py:198
msgid "cant_open_a_channel_with_only_yourself"
msgstr "Vous ne pouvez pas ouvrir un canal avec seulement vous-même."

#: messaging/resources/messaging.py:221
msgid "channel_can_only_be_opened_between_a_member_and_a_practitioner"
msgstr "Le canal ne peut être ouvert qu'entre un membre et un praticien."

#: messaging/resources/messaging.py:495
msgid "could_not_find_asset_matching_requested_identifier"
msgstr "Impossible de trouver l'actif correspondant à l'identifiant demandé, {asset_id}."

#: messaging/resources/messaging.py:597
msgid "exception_allocating_message_credits_enterprise_user"
msgstr "Vous n'avez pas assez de crédit pour envoyer des messages en tant qu'utilisateur entreprise."

#: messaging/resources/messaging.py:605
msgid "exception_allocating_message_credits_marketplace_user"
msgstr "Vous n'avez pas assez de crédit pour envoyer un message."

#: messaging/resources/messaging.py:643
msgid "general_exception_allocating_message_credits"
msgstr "Exception lors de l'allocation des crédits de message"

#: messaging/resources/sms.py:123
msgid "generic_maven_message_body"
msgstr "Vous avez un nouveau message de Maven!"

#: messaging/resources/messaging.py:131
msgid "input_error_getting_channels"
msgstr "Erreur d'entrée lors de la récupération des données des canaux"

#: messaging/resources/messaging.py:464
msgid "message_is_too_long"
msgstr "Votre message est trop long! Veuillez le réduire à un maximum de {max_n_chars} caractères."

#: messaging/schemas/messaging.py:34
msgid "message_reply_sla_user_message_ca_default"
msgstr "Vous recevrez une réponse sous 24 heures."

#: messaging/schemas/messaging.py:37
msgid "message_reply_sla_user_message_provider_default"
msgstr "La plupart des spécialistes répondent dans les 24 heures."

#: messaging/schemas/messaging.py:40
msgid "message_reply_sla_user_message_wallet"
msgstr "Vous recevrez une réponse sous 3 jours ouvrables."

#: messaging/schemas/messaging.py:44
msgid "message_reply_sla_user_message_wallet_with_inbound_phone_number"
msgstr "Vous recevrez une réponse sous 3 jours ouvrables. Si votre demande est urgente, veuillez plutôt nous appeler au {phone_number}."

#: messaging/resources/messaging.py:70
msgid "no_card_on_file_error"
msgstr "Le client n'a pas de carte enregistrée."

#: messaging/resources/messaging.py:213
msgid "one_or_more_users_not_found"
msgstr "Un ou plusieurs utilisateurs introuvables."

#: messaging/resources/messaging.py:578
msgid "only_image_and_pdf_file_types_are_supported"
msgstr "Seuls les types de fichiers image et PDF sont pris en charge."

#: messaging/resources/messaging.py:179
msgid "serializing_error_getting_channels"
msgstr "Erreur de sérialisation lors de l'obtention des données des canaux"

#: messaging/resources/messaging.py:431
msgid "unable_to_post_message_in_channel_inactive_or_messaging_disabled_practitioner"
msgstr "Impossible de publier un message dans le canal {channel_id} : praticien inactif ou messagerie désactivée"

#: messaging/resources/messaging.py:456
msgid "unable_to_post_message_in_channel_member_cannot_message_practitioner"
msgstr "Impossible de publier le message, le membre ne peut pas envoyer de message à ce spécialiste."

#: messaging/resources/messaging.py:374
msgid "unable_to_post_message_in_channel_no_participants_found"
msgstr "Impossible de publier un message dans le canal {channel_id} : aucun participant trouvé"

#: messaging/resources/messaging.py:395
msgid "unable_to_post_message_in_channel_user_is_not_a_participant"
msgstr "Impossible de publier un message dans le canal {channel_id} : l'utilisateur n'est pas un participant"

#: messaging/resources/messaging.py:208
msgid "we_dont_support_multi_party_messaging"
msgstr "Nous ne prenons pas en charge la messagerie multi-parties."

#: messaging/resources/messaging.py:203
msgid "you_are_already_included_in_the_channel"
msgstr "Vous êtes déjà inclus(e) dans le canal, il n'est donc pas nécessaire de vous inclure dans les user_ids."
